apps/frontend/readme.md

```diff
@@ -3,9 +3,47 @@
 ### Requires node 17
 
 ### Install deps
+
 `yarn install`
 
+### Login with Google Cloud with @nav.no user
+
+Do this in Google Chrome as it doesn't work in Firefox
```

Comment:
Hmm, it works on my machine (in Firefox), although maybe that's Linux magic at play?